crates/blockchain/blockchain.rs

```diff
@@ -43,15 +42,10 @@ pub fn add_block(block: &Block, storage: &Store) -> Result<(), ChainError> {
     // Validate the block pre-execution
     validate_block(block, &parent_header, &state)?;
     let (receipts, account_updates): (Vec<Receipt>, Vec<AccountUpdate>) = {
-        match EVM_BACKEND.get() {
-            Some(EVM::LEVM) => backends::levm::execute_block(block, &mut state)?,
-            // This means we are using REVM as default for tests
-            Some(EVM::REVM) | None => {
-                let receipts = backends::revm::execute_block(block, &mut state)?;
-                let account_updates = ethrex_vm::get_state_transitions(&mut state);
-                (receipts, account_updates)
-            }
-        }
+        EVM_BACKEND
+            .get()
+            .unwrap_or(&EVM::default())
+            .execute_block(block, &mut state)?
```

Comment:
I think you probably want [`get_or_init`](https://doc.rust-lang.org/std/sync/struct.OnceLock.html#method.get_or_init) here:
```suggestion
        EVM_BACKEND
            .get_or_init(EVM::default)
            .execute_block(block, &mut state)?
```